packages/labs/virtualizer/src/support/method-interception.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type Function = (...args: any[]) => any;
```

Comment:
There's a built in type with this name that also has roughly this type I think, could you use that?

Reply:
It apparently doesn't like it for `Parameters<T>` where `T extends Function`

Maybe I'll rename it something that doesn't consume the name or cause confusion.  `AnyFunction`?

Reply:
Renamed to `AnyFunction`. Apparently `Parameters<T>` requires `T` to extend `(...args: any[]) => any`